drivers/hmis_external_apis/app/models/hmis_external_apis/sh_hmis/importers/loaders/youth_education_status_loader.rb

```diff
@@ -0,0 +1,163 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# reload!; reader = HmisExternalApis::ShHmis::Importers::Loaders::CsvReader.new('drivers/hmis_external_apis/spec/fixtures/hmis_external_apis/sh_hmis/importers'); HmisExternalApis::ShHmis::Importers::Loaders::YouthEducationStatusLoader.new(clobber: false, reader: reader).perform
+module HmisExternalApis::ShHmis::Importers::Loaders
+  class YouthEducationStatusLoader < SingleFileLoader
+    # Key so that we know which YouthEducationStatus were generated by this process, so that we can clobber them
+    SOURCE_HASH = 'OP-HMIS-MIGRATION-2023'.freeze
```

Comment:
@eanders is this an OK thing to do? The idea is that when we re-run this we delete the old records, so we need to be able to identify them.

Reply:
That's probably fine given that we won't ever import an HMIS CSV on top of it, and if we did it will think the data has changed so should overwrite it.